src/brandbar.module.css

```diff
@@ -0,0 +1,13 @@
+.dropdownItem {
+  padding: 8px;
+  color: white;
+}
+
+.dropdownItem:hover {
+  color: black;
+}
+
+.dropdownToggle {
+  display: flex;
+  align-items: center;
+}
```

Comment:
This shouldn't be necessary.  If you have the webapp's loading their styles from the dev landing page, they should already be picking up this setting from `branding.css`.